packages/widgetbook/lib/src/app/router.dart

```diff
@@ -24,51 +24,32 @@ GoRouter createRouter({
 }) {
   return GoRouter(
     initialLocation: initialLocation,
-    redirect: (context, state) {
-      // Redirect deprecated `disable-navigation` and `disable-properties`
-      // query params to their equivalent `panels` query param.
-      if (state.queryParams.containsKey('disable-navigation') ||
-          state.queryParams.containsKey('disable-properties')) {
-        final disableNavigation = _parseBoolQueryParameter(
-          value: state.queryParams['disable-navigation'],
-        );
-
-        final disableProperties = _parseBoolQueryParameter(
-          value: state.queryParams['disable-properties'],
-        );
-
-        final panels = {
-          if (!disableNavigation) ...{
-            WidgetbookPanel.navigation,
-          },
-          if (!disableProperties) ...{
-            WidgetbookPanel.addons,
-            WidgetbookPanel.knobs,
-          }
-        };
-
-        return '/?panels={${panels.map((x) => x.name).join(",")}}';
-      }
-
-      return null;
-    },
     routes: [
       ShellRoute(
-        builder: (_, state, child) => WidgetbookScope(
-          state: WidgetbookState(
-            path: state.queryParams['path'] ?? '',
-            panels: state.queryParams['panels'] == null
-                ? WidgetbookPanel.values.toSet()
-                : WidgetbookPanelParser.parse(state.queryParams['panels']!),
-            queryParams: {...state.queryParams}, // Copy from UnmodifiableMap
-            addons: addons,
-            appBuilder: appBuilder,
-            catalog: catalog,
-          ),
-          child: WidgetbookShell(
-            child: child,
-          ),
-        ),
+        builder: (_, state, child) {
+          // If `preview` does not exist, and any of the deprecated query params
+          // `disable-navigation` or `disable-properties` exist with `true`,
+          // then preview mode will be enabled.
+          final isPreview = state.queryParams.containsKey('preview') ||
+              state.queryParams.getBool('disable-navigation') ||
+              state.queryParams.getBool('disable-properties');
```

Comment:
Do we need to support the old versions as well? I think we should change the cloud in a way that it'll set both parameters and parameter since parameters that are unknown are simply ignored.

Reply:
I've removed support for the `disable-*` params, now it's essential to set them in the cloud to support older versions of Widgetbook.